policies/verify-instances-templates-reference/README.md

```diff
@@ -0,0 +1,59 @@
+VERIFY INSTANCES TEMPLATES REFERENCE
+=====================================
+## GOALS 
+This policy verifies that an instance  refers to an existing template in the correct namespace when it is created or updated.
+
+## TESTS
+Tests are available in folder [policies](./policies).
+
+## HOW TO RUN
+Run the following commands inside folder [manifest](./manifest):
+- kubectl create -f config-sync.yaml
+- kubectl create -f constraintTemplate.yaml
+- kubectl create -f constraint.yaml
+
+## Policies
+
+**Severity:** Violation
+
+**Resources:** crownlabs.polito.it/Instance[enter link description here](../../operators/deploy/crds/crownlabs.polito.it_instances.yaml)
+
+
+
+### Rego
```

Comment:
I personally would avoid to have code in documentation... The risk is that it easily become not updated with the code in the repo.